scripts/sanitizeTranslations.js

```diff
@@ -0,0 +1,94 @@
+/*eslint-disable  @typescript-eslint/no-var-requires*/
```

Comment:
This is a sanitize script that is used on the translations after it is downloaded.